src/main/features/boot.json

```diff
@@ -84,6 +84,14 @@
             "id":"org.apache.felix:org.apache.felix.log:1.2.6",
             "start-order":"1"
         },
+        {
+            "id":"org.eclipse.parsson:jakarta.json:1.1.1",
```

Comment:
What is this artifact? Shouldn't we rather reference the official API dependency, https://search.maven.org/artifact/jakarta.json/jakarta.json-api/2.1.1/jar according to https://jakarta.ee/specifications/jsonp/2.1/

Reply:
Ok, I see now this is the actual JSON P implementation (not the API), maybe adding a comment what this artifact is would help.

Reply:
Also I think we still need https://mvnrepository.com/artifact/jakarta.json/jakarta.json-api (in addition)

Reply:
Ok, it seems that basically `org.eclipse.parsson:jakarta.json` is an aggregate of `org.eclipse.parsson:parsson:1.1.1` and `jakarta.json:jakarta.json-api`. I think we should rather rely on the individual bundles instead of the aggregate